rllib/tuned_examples/ppo/pendulum-ppo.yaml

```diff
@@ -17,7 +17,6 @@ pendulum-ppo:
         lr: 0.0003
         sgd_minibatch_size: 64
         observation_filter: MeanStdFilter
-        seed: 42
```

Comment:
This is not needed to get things green but we should definitely not rely on a seed to make rllib/tuned_examples/ppo/pendulum-ppo.yaml learn pendulum consistenly 😄